packages/peregrine/lib/talons/Header/useCartTrigger.js

```diff
@@ -42,16 +45,21 @@ export const useCartTrigger = props => {
         }
     }, [cartId, getItemCount]);
 
-    const handleClick = useCallback(async () => {
-        toggleDrawer('cart');
```

Comment:
Instead of removing the callback we should add a deprecation notice and add a feature toggle. This would allow us to gracefully handle removing the old mini cart code, would allow us to release `develop` without worrying about the new mini cart being complete, and would allow external partners using `develop` to decide when to switch.

Reply:
Resolved in https://github.com/magento/pwa-studio/pull/2494/commits/6524bd78dd6a6836ac1d3684ae6d9e5180d3265f.